app/src/main/java/org/mozilla/focus/fragment/ListPanelDialog.java

```diff
@@ -28,6 +28,7 @@
     private View downloadsTouchArea;
     private View historyTouchArea;
     private View screenshotsTouchArea;
+    private int type;
```

Comment:
We don't need this, we can use getArguments().getInt(TYPE). getArguments().getInt(TYPE) is also more promising if the Fragment is recycled and recreated by the system.